src/core/namespace.c

```diff
@@ -632,6 +632,7 @@ static int mount_private_dev(MountEntry *m) {
         return 0;
 
 fail:
+        log_warning_errno(r, "Failed to setup private /dev, ignoring: %m");
```

Comment:
This does not look correct, not all failure paths go through `fail`. This seems to be the wrong place. Why not do the print in the caller?